Source/Reporters/JunitXMLReporter.jsl

```diff
@@ -159,13 +164,10 @@ Define Class(
         test_case_xml ||= this:generate_xml_tag("testcase", trim(test_case_content), props, 0);
         
       );
-      num_tests = NItems(test_case_names);
       props = Associative Array();
       props["name"] = test_suites[i];
       props["tests"] = Char(num_tests);
-
-      // find test results that are non empty
-      num_successes = Nitems(loc(test_case_info, [=>]));
+      show(num_successes);
```

Comment:
Remove `show` please.

Reply:
done in f91cfdac472b4c83635cb97fdb4b1a510082cd3a